pages/createPlan/index.js

```diff
@@ -1,9 +1,14 @@
 import PlanForm from "@/components/PlanForm/PlanForm";
+import styled from "styled-components";
+
+const StyledFormContainer = styled.section`
+  min-height: 100vh;
```

Comment:
I don't understand the purpose here: why do you need this HTML element with only the `height` set? Why not passing the `height` to the form directly (if necessary)?

And in which cases is the container higher than `100vh`? And does the user realizes this?

Reply:
Honestly, my head's currently spinning. The navbar acted up - it was covering a small portion of content - and this fixes it. Same with all other height rule related suggestions. I'm sure there is a better way. I can't even comprehend your suggestion right now, but will look into it.